packages/trpc/migrations/migration.json

```diff
@@ -1,4 +1,16 @@
 {
   "$schema": "../../../node_modules/@angular-devkit/schematics/collection-schema.json",
-  "schematics": {}
+  "schematics": {},
+  "packageJsonUpdates": {
+    "0.2.30": {
```

Comment:
Question - do we need to define additional updates to handle the `1.0.0-beta` versions?

Also, let me know if these targeted versions are correct. Based them off the latest release version.

Reply:
No, we don't need to target the beta versions also.